app/javascript/react/components/Square.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import LightSquareImage from "../../../assets/images/light-square.png"
+import DarkSquareImage from "../../../assets/images/dark-square.png"
```

Comment:
using images for squares is fine for now, but in the long run it will likely be easier if you render a div that is STYLED to look like a light or dark square, because you'll have to overlay the image of a piece at some point. Otherwise, you would need an image of every single piece in both types of squares later on